<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1629</width>
    <height>855</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(41, 152, 255);
background-color: rgb(0, 0, 0);</string>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>1371</width>
      <height>781</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>HOME</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>-50</x>
        <y>-10</y>
        <width>1441</width>
        <height>791</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../Pictures/Screenshots/Screenshot 2025-01-18 135540.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>-170</x>
        <y>90</y>
        <width>821</width>
        <height>131</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Playbill</family>
        <pointsize>30</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 8pt &quot;Terminal&quot;;
color: rgb(8, 8, 8);
background-color: rgba(255, 255, 255, 0);
font: 47pt &quot;Terminal&quot;;
font: 30pt &quot;Playbill&quot;;
</string>
      </property>
      <property name="text">
       <string>HEY THERE...! WELCOME</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="REC">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>230</y>
        <width>301</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#REC{
	border-radius:20px;
	border-bottom:2px solid rgba(0,0,0,100);
	color: rgb(0, 170, 255);
	color: rgb(39, 39, 39);
	font: 23pt &quot;Terminal&quot;;
	background-color: rgb(220, 222, 255);
	padding:10px;
}

QPushButton#REC:hover{
	color: rgb(0, 170, 255);
	color: rgb(39, 39, 39);
	background-color: rgb(157, 159, 157);
}</string>
      </property>
      <property name="text">
       <string>Recruiting</string>
      </property>
     </widget>
     <widget class="QPushButton" name="REC_2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>370</y>
        <width>301</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#REC_2{
	border-radius:20px;
	border-bottom:2px solid rgba(0,0,0,100);
	color: rgb(0, 170, 255);
	color: rgb(39, 39, 39);
	font: 23pt &quot;Terminal&quot;;
	background-color: rgb(220, 222, 255);
	padding:10px;
}

QPushButton#REC_2:hover{
	color: rgb(0, 170, 255);
	color: rgb(39, 39, 39);
	background-color: rgb(157, 159, 157);
}</string>
      </property>
      <property name="text">
       <string>Chat with us...!</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>RECRUITMENT</string>
     </attribute>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1371</width>
        <height>751</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>26</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../Pictures/Screenshots/Screenshot 2025-01-18 143116.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>61</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 8pt &quot;Terminal&quot;;
color: rgb(190, 184, 177);
background-color: rgba(255, 255, 255, 0);
font: 47pt &quot;Terminal&quot;;
font: 20pt &quot;Playbill&quot;;
</string>
      </property>
      <property name="text">
       <string>NAME:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>90</y>
        <width>351</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit#lineEdit{
	background-color: rgb(188, 208, 197);
	border-radius:20px;
	border-bottom:2px solid rgba(0,0,0,100);
	padding:10px;
	font:14pt&quot;Terminal&quot;;
	color: rgb(25, 25, 25);
}

QLineEdit#lineEdit:hover{
	background-color: rgb(188, 208, 197);
	background-color: rgb(142, 153, 137);
	color: rgb(25, 25, 25);
}</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>200</y>
        <width>81</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 8pt &quot;Terminal&quot;;
color: rgb(190, 184, 177);
background-color: rgba(255, 255, 255, 0);
font: 47pt &quot;Terminal&quot;;
font: 20pt &quot;Playbill&quot;;
</string>
      </property>
      <property name="text">
       <string>AGE:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>190</y>
        <width>231</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit#lineEdit_2{
	background-color: rgb(188, 208, 197);
	border-radius:20px;
	border-bottom:2px solid rgba(0,0,0,100);
	padding:10px;
	font:14pt&quot;Terminal&quot;;
	color: rgb(25, 25, 25);
}

QLineEdit#lineEdit_2:hover{
	background-color: rgb(188, 208, 197);
	background-color: rgb(142, 153, 137);
	color: rgb(25, 25, 25);
}</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>290</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 8pt &quot;Terminal&quot;;
color: rgb(190, 184, 177);
background-color: rgba(255, 255, 255, 0);
font: 47pt &quot;Terminal&quot;;
font: 20pt &quot;Playbill&quot;;
</string>
      </property>
      <property name="text">
       <string>EXPERIENCE:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>280</y>
        <width>131</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit#lineEdit_3{
	background-color: rgb(188, 208, 197);
	border-radius:20px;
	border-bottom:2px solid rgba(0,0,0,100);
	padding:10px;
	font:14pt&quot;Terminal&quot;;
	color: rgb(25, 25, 25);
}

QLineEdit#lineEdit_3:hover{
	background-color: rgb(188, 208, 197);
	background-color: rgb(142, 153, 137);
	color: rgb(25, 25, 25);
}</string>
      </property>
     </widget>
     <widget class="QPushButton" name="BACK">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>410</y>
        <width>151</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Terminal</family>
        <pointsize>23</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#BACK{
	border-radius:20px;
	border-bottom:2px solid rgba(0,0,0,100);
	color: rgb(0, 170, 255);
	color: rgb(39, 39, 39);
	font: 23pt &quot;Terminal&quot;;
	background-color: rgb(220, 222, 255);
	padding:10px;
}

QPushButton#BACK:hover{
	color: rgb(0, 170, 255);
	color: rgb(39, 39, 39);
	background-color: rgb(157, 159, 157);
}</string>
      </property>
      <property name="text">
       <string>BACK</string>
      </property>
     </widget>
     <widget class="QPushButton" name="BACK_2">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>410</y>
        <width>161</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Terminal</family>
        <pointsize>23</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#BACK_2{
	border-radius:20px;
	border-bottom:2px solid rgba(0,0,0,100);
	color: rgb(0, 170, 255);
	color: rgb(39, 39, 39);
	font: 23pt &quot;Terminal&quot;;
	background-color: rgb(220, 222, 255);
	padding:10px;
}

QPushButton#BACK_2:hover{
	color: rgb(0, 170, 255);
	color: rgb(39, 39, 39);
	background-color: rgb(157, 159, 157);
}</string>
      </property>
      <property name="text">
       <string>ENTER</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>SCHEDULING</string>
     </attribute>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>-100</x>
        <y>0</y>
        <width>1521</width>
        <height>841</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../hackoholic/WhatsApp Image 2025-01-20 at 13.55.14_c5fed3a8.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>80</y>
        <width>421</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Playbill</family>
        <pointsize>30</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 8pt &quot;Terminal&quot;;
color: rgb(8, 8, 8);
background-color: rgba(255, 255, 255, 0);
font: 47pt &quot;Terminal&quot;;
font: 30pt &quot;Playbill&quot;;
</string>
      </property>
      <property name="text">
       <string>HOW CAN I HELP YOU..?</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="REC_3">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>360</y>
        <width>301</width>
        <height>101</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Terminal</family>
        <pointsize>23</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#REC_3{
	border-radius:20px;
	border-bottom:2px solid rgba(0,0,0,100);
	color: rgb(0, 170, 255);
	color: rgb(39, 39, 39);
	font: 23pt &quot;Terminal&quot;;
	background-color: rgb(220, 222, 255);
	padding:10px;
}

QPushButton#REC_3:hover{
	color: rgb(0, 170, 255);
	color: rgb(39, 39, 39);
	background-color: rgb(157, 159, 157);
}</string>
      </property>
      <property name="text">
       <string>GIVE ME ANSWER....</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>190</y>
        <width>511</width>
        <height>141</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit#lineEdit_4{
	background-color: rgb(188, 208, 197);
	border-radius:20px;
	border-bottom:2px solid rgba(0,0,0,100);
	padding:10px;
	font:14pt&quot;Terminal&quot;;
	color: rgb(25, 25, 25);
}

QLineEdit#lineEdit_4:hover{
	background-color: rgb(188, 208, 197);
	background-color: rgb(142, 153, 137);
	color: rgb(25, 25, 25);
}</string>
      </property>
     </widget>
     <widget class="QPushButton" name="BACK_3">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>500</y>
        <width>151</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Terminal</family>
        <pointsize>23</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#BACK_3{
	border-radius:20px;
	border-bottom:2px solid rgba(0,0,0,100);
	color: rgb(0, 170, 255);
	color: rgb(39, 39, 39);
	font: 23pt &quot;Terminal&quot;;
	background-color: rgb(220, 222, 255);
	padding:10px;
}

QPushButton#BACK_3:hover{
	color: rgb(0, 170, 255);
	color: rgb(39, 39, 39);
	background-color: rgb(157, 159, 157);
}</string>
      </property>
      <property name="text">
       <string>BACK</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>1540</x>
      <y>90</y>
      <width>211</width>
      <height>801</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>81</width>
      <height>831</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>810</y>
      <width>1531</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>1470</x>
      <y>20</y>
      <width>241</width>
      <height>781</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>41</width>
      <height>781</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>780</y>
      <width>1411</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>1621</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
